packages/next-app/src/app/metadata/(components)/OwnedPoolsSidebarItems.tsx

```diff
@@ -91,25 +130,41 @@ function PoolCard({ isSelected, pool }: { isSelected: boolean; pool: Pool }) {
 
   const poolName =
     poolType === "Weighted" && tokens
-      ? tokens.map((obj) => obj.symbol).join("/")
+      ? tokens.map((obj) => obj.symbol).join(" / ")
       : name;
   const weights =
     poolType === "Weighted" && tokens
-      ? tokens.map((obj) => (Number(obj.weight) * 100).toFixed()).join("/")
+      ? tokens
+          .sort((a, b) => (a.weight < b.weight ? 1 : -1))
```

Comment:
What's the rationale behind sorting this? Does Balancer do it on their frontend? If not I'd suggest we keep to the same order returned form the subgraph